packages/shared/src/components/MainLayout.tsx

```diff
@@ -19,6 +19,9 @@ import styles from './MainLayout.module.css';
 import ProfileButton from './profile/ProfileButton';
 import { LinkWithTooltip } from './tooltips/LinkWithTooltip';
 import Sidebar from './sidebar/Sidebar';
+import MenuIcon from '../../icons/filled/hamburger.svg';
```

Comment:
Do note: We don't have a valid system for the new system, and currently we have to support the old and new ones for the time being.

We might want to refactor icon usage after relayout!

Reply:
When refactoring icon usage we might need to think again about SVGR.
I heard there are better ways now to handle icons.
SVGR can be memory intensive.